packages/devtools-reps/src/reps/tests/error.js

```diff
@@ -334,6 +334,18 @@ describe("Error - base-loader.js", () => {
   });
 });
 
+describe("Error - longString stacktrace", () => {
+  const stub = stubs.get("longString stack Error");
+
+  it("renders as expected", () => {
+    const renderedComponent = shallow(ErrorRep.rep({
+      object: stub
+    }));
+
+    expect(renderedComponent).toMatchSnapshot();
```

Comment:
The test should also cover case where the stack is complete, ie it's string not long string

Reply:
Ok, excellent.

Thanks!

Honza